fsm/fsm/trailbot_fsm.py

```diff
@@ -46,15 +46,20 @@ def execute(self, blackboard):
         return 'not_arrived'
 
 class WaitState(State):
-    def __init__(self, state_publisher, blackboard):
+    def __init__(self, query_publisher, state_publisher, blackboard):
```

Comment:
query_publisher is not needed. Voice assistant node will subscribe to the state_publisher